### ISSUE TITLE
cbslocal.com | Breakage

### ISSUE THREAD

**Issue by in4u** [uBlockOrigin/uAssets#6866]
### URL(s) where the issue occurs

`https://sanfrancisco.cbslocal.com/video/4413737-furries-stop-assault-of-woman-on-san-jose-streets/`

### Describe the issue

The video is blocked.

### Versions

- Browser/version: Firefox 72.0.2
- uBlock Origin version: 1.24.2

### Settings

Default

### Notes

Disabling `EasyPrivacy` and `EasyList Cookie` (I don't remember adding latter manually nor do I see it listed as part of [uBO default](https://github.com/gorhill/uBlock/wiki/Blocking-mode#easy-mode-details-), has it been added recently?) resolves the issue.

**Comment by okiehsch** [comment]
> EasyList Cookie (I don't remember adding latter manually nor do I see it listed as part of uBO default, has it been added recently?)

No, that list is not enabled by default.
````
@@||cdn.cookielaw.org/scripttemplates/*$script,xhr,domain=sanfrancisco.cbslocal.com
@@||cdn.cookielaw.org/consent/*$xhr,domain=sanfrancisco.cbslocal.com
`````
fixes the issue, or simply disable `EasyList Cookie`.

**Comment by yourduskquibbles** [comment]
Video is broken for me on `https://losangeles.cbslocal.com/live/cbsn-la/` with default setup. Video loads fine on no extension profile.

**Comment by yourduskquibbles** [comment]
`@@||production-cmp.isgprivacy.cbsi.com/dist/optanon.js$script,domain=cbslocal.com` fixes for me, I will add to `uBlock filters – Unbreak​​​​​`

**Comment by mapx-** [comment]
@ryanbr

**Comment by llacb47** [comment]
the identical filter exists one line above, why not add another domain as exception

**Comment by yourduskquibbles** [comment]
I realize that (and why I placed it there in the list) but that doesn't seem to be standard practice throughout other areas of the list when it's separate domains/issues being fixed (e.g. search for `@@||googletagmanager.com/gtm.js$script,domain=` or `@@||google-analytics.com/analytics.js$script,domain=` so I just placed next to the other line. For list file size, it would be better to just consolidate & group the filters but I don't know the history on why filters aren't grouped together when they could be like in this instance.
